io/disk/multipath_test.py

```diff
@@ -119,8 +119,7 @@ def test(self):
         # Print errors
         if msg:
             msg = "Following Tests Failed\n" + msg
-            self.log.debug(msg)
-            self.fail("Some tests failed")
+            self.fail("Some tests failed. Find detaild below:\n%s", msg)
```

Comment:
@PraveenPenguin It got this suggestion from community which I shared here, I am okay.